dtreeviz/trees.py

```diff
@@ -148,7 +148,8 @@ def ctree_leaf_distributions(self,
                                  fontname: str = "Arial",
                                  grid: bool = False,
                                  figsize: tuple = None,
-                                 ax=None):
+                                 ax=None,
```

Comment:
It's weird that your PR is pulling these in because this is from one of our earlier changes. Maybe you created this PR  or branch before pulling the latest from dev?

Reply:
I'm not sure what you mean, but I noticed that the param doc text for figsize and ax did not have the correct format, so I fixed that at the same time.

I will move `label_all_leaves` up!

Reply:
oh, now it is looking clean. weird. Maybe I needed to do a refresh of the webpage.

Reply:
I did a rebase to the current dev branch with that last commit.